stable/pachyderm/templates/NOTES.txt

```diff
@@ -0,0 +1,15 @@
+In order to use Pachyderm, you need to install the Pachyderm command line client: pachctl.
+
+On Linux environments:
+
+$ curl -o /tmp/pachctl.deb -L https://github.com/pachyderm/pachyderm/releases/download/v1.6.6/pachctl_1.6.6_amd64.deb && sudo dpkg -i /tmp/pachctl.deb
```

Comment:
with appVersion set in Chart.yaml, you can template this with `{{.Chart.AppVersion}}` I think